pkg/sql/catalog/catpb/privilege.go

```diff
@@ -462,10 +465,26 @@ type UserPrivilege struct {
 
 // Show returns the list of {username, privileges} sorted by username.
 // 'privileges' is a string of comma-separated sorted privilege names.
-func (p PrivilegeDescriptor) Show(objectType privilege.ObjectType) []UserPrivilege {
+// The owner always implicitly receives ALL privileges with the GRANT OPTION. If
+// showImplicitOwnerPrivs is true, then that implicit privilege is shown here.
```

Comment:
Would it make sense to define a `ShowWithoutImplicitPrivs` that wraps a base `show` function such that `Show` continues to do what we currently do, `ShowWithOutImplicitPrivs` is called for the (I think) one bulkio case?

just to reduce churn here and having to always do ` true /* showImplicitOwnerPrivs */`

Reply:
i'm on the fence.. if there are more bulk-io use cases in the future, then it won't be obvious to the code writer that there's  a `ShowWithoutImplicitPrivs` function

in general, i actually don't anticipate this function being called in that many places, so i think it's fine to be explicit with a bit more mental overhead.

Reply:
Yeah that's reasonable